lib/ads/interstitial_ad_manager.js

```diff
@@ -1352,8 +1358,6 @@ shaka.ads.InterstitialAdManager = class {
     this.player_.configure('ads.disableDASHInterstitial', true);
     const netEngine = this.player_.getNetworkingEngine();
     goog.asserts.assert(netEngine, 'Need networking engine');
-    netEngine.clearAllRequestFilters();
```

Comment:
Why are these removed?  Is this related to exposing the interstitial player?

Reply:
Yes, it's not longer necessary